src/features/ValueEditor/ValueEditor.tsx

```diff
@@ -49,31 +107,31 @@ const editorOptions: editor.IStandaloneEditorConstructionOptions = {
 }
 
 interface Props {
-  validation: Validation
+  schema?: ZodSchema<any, z.ZodTypeDef, any> | undefined
```

Comment:
`undefined` is not needed here.

Reply:
if we want to nitpick here, `?` is the one that it's not needed because a prop is always passed, and its value can be undefined.

The `?` instead, also allows for passing no `schema` prop at all